### PR TITLE
added strategy documentation

### PR DESCRIPTION
**Pull request by gowerc**

### CODE REVIEW

File: R/strategies.R [hunk 1]
```diff
@@ -1,16 +1,27 @@
...
 #' Reference (CIR) and Jump To Reference (JTR) methods, to adapt the covariance matrix
-#' to different pre-deviation and post deviation covariance structures.
+#' to different pre-deviation and post deviation covariance structures. See Carpenter
+#' et al. (2013)
 #'
-#' @param sigma_group TODO
-#' @param sigma_ref TODO
-#' @param index_mar TODO
+#' @param sigma_group the covariance matrix with dimensions equal to `index_mar` for
+#' the subjects original group
```
Comment (nociale) [inline R/strategies.R:13]: "the covariance matrix with dimensions equal to the length of `index_mar` for the subjects treatment group"

File: R/strategies.R [hunk 2]
```diff
@@ -1,16 +1,27 @@
...
+#' @param sigma_ref the covariance matrix with dimensions equal to `index_mar` for
+#' the subjects reference group
+#' @param index_mar A logical vector indicating which visits meet the MAR assumption
+#' for the subject. I.e. this identifies the observations that after a non-MAR
```
Comment (nociale) [inline R/strategies.R:17]: I.e. this identifies the observations that after a non-MAR intercurrent event (ICE). -> I.e. this identifies the observations after a non-MAR intercurrent event (ICE).

File: R/strategies.R [hunk 3]
```diff
@@ -55,29 +66,53 @@ compute_sigma <- function(sigma_group, sigma_ref, index_mar){
...
+#' strategies by combining a subjects own distribution with that of
+#' a reference distribution based upon which of their visits failed to meet
+#' the MAR assumption
+#'
+#' @param pars_group A list of parameters for the subject's group. See details.
+#'
+#' @param pars_ref A list of parameters for the subject's reference group. See details.
+#'
+#' @param index_mar A logical vector indicating which visits meet the MAR assumption
+#' for the subject. I.e. this identifies the observations that after a non-MAR
```
Comment (nociale) [inline R/strategies.R:82]: without "that"

File: R/strategies.R [hunk 4]
```diff
@@ -55,29 +66,53 @@ compute_sigma <- function(sigma_group, sigma_ref, index_mar){
...
+#'
+#' @details
+#'
+#' `pars_group` and `pars_ref` both must be a list containing elements `mu` and `sigma`.
+#' `mu` must be a numeric vector and `sigma` must be a square matrix symmetric covariance
+#' matrix with dimensions equal to the length of `mu` and `index_mar`. e.g.
+#' ```
+#' list(
+#'     mu = c(1,2,3),
+#'     sigma = matrix(c(1,2,3,4,5,6,7,8,9), nrow = 3, ncol = 3)
```
Comment (nociale) [inline R/strategies.R:93]: is this sigma symmetric?

File: R/strategies.R [hunk 5]
```diff
@@ -55,29 +66,53 @@ compute_sigma <- function(sigma_group, sigma_ref, index_mar){
...
+#' - Missing at Random (MAR)
+#' - Jump to Reference (JR)
+#' - Copy Reference (CR)
+#' - Copy Increments from Reference (CIR)
+#' - Last Mean Carried Forward (LMCF)
 #'
-#' @param pars_group TODO
-#' @param pars_ref TODO
-#' @param index_mar TODO
+#' @name stratgies
```
Comment (nociale) [inline R/strategies.R:107]: I see you called it "stratgies" instead of "strategies". Is this what you want?

File: R/strategies.R [hunk 6]
```diff
@@ -189,27 +203,30 @@ strategy_LMCF <- function(pars_group, pars_ref, index_mar){
...
 #' The user can define their own strategy functions (or overwrite the pre-defined ones)
 #' by specifying a named input to the function i.e. `NEW = function(...) ...`.
 #' Only exception is MAR which cannot be overwritten
 #'
-#' All user defined functions must take 3 inputs `pars_group`, `pars_ref` & `index_mar`.
-#' `pars_group` and `pars_ref` are both lists with elements `mu` and `sigma` representing
-#' the multivariate normal distribution parameters for the subjects current and reference group
-#' respectively.  `index_mar` will be a logical vector specifying which visits the subject met
-#' the MAR assumption at. The function must return a list with elements `mu` and `sigma`.
+#' All user defined functions must take 3 inputs `pars_group`, `pars_ref` &
```
Comment (nociale) [inline R/strategies.R:220]: All user defined functions must take 3 inputs `pars_group` -> All user defined functions must take 3 inputs: `pars_group`

File: R/strategies.R (hunk 1 above)
Comment (gowerc) [reply R/strategies.R:13]: But its not necessarily a treatment group is it ?

File: R/strategies.R (hunk 2 above)
Comment (gowerc) [reply R/strategies.R:17]: Done

File: R/strategies.R (hunk 4 above)
Comment (gowerc) [reply R/strategies.R:93]: it is not, fixed

File: R/strategies.R (hunk 5 above)
Comment (gowerc) [reply R/strategies.R:107]: typo. Fixed

File: R/strategies.R (hunk 6 above)
Comment (gowerc) [reply R/strategies.R:220]: done

File: R/strategies.R (hunk 3 above)
Comment (gowerc) [reply R/strategies.R:82]: I had actually missed out the word occur,  have replaced it with: 

> this identifies the observations that --occur-- after a non-MAR intercurrent event (ICE)

File: R/strategies.R (hunk 1 above)
Comment (nociale) [reply R/strategies.R:13]: You're right :).

**Comment by gowerc** [comment]
@nociale , can this be merged then /

**Comment by nociale** [comment]
> @nociale , can this be merged then /

Yes :)

**Pull request merged**